packages/mdx/test/index.test.js

```diff
@@ -257,7 +258,9 @@ describe('@mdx-js/mdx', () => {
   })
 
   it('should support tables in markdown', async () => {
-    const Content = await run('| A | B |\n| :- | -: |\n| a | b |')
+    const Content = await run('| A | B |\n| :- | -: |\n| a | b |', {
+      remarkPlugins: [gfm]
+    })
```

Comment:
micromark/remark is commonmark compliant, and GFM (strikethrough, tables, literal URLs, tasklists) are enabled by using a plugin